### ISSUE TITLE
Latent , 2 lora , Hires.fix  =  KeyError 

### ISSUE THREAD

**Issue by freebag668** [hako-mikan/sd-webui-regional-prompter#142]
![screencapture-127-0-0-1-7860-2023-06-06-01_20_31](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/129811700/677598fc-f57d-41f9-9dce-cb09445b1e20)

Extension Version [f71aa479](https://github.com/hako-mikan/sd-webui-regional-prompter/commit/f71aa479e00736e2c9e19d986f977bdb4bf4f9c7)

<img width="1920" alt="001" src="https://github.com/hako-mikan/sd-webui-regional-prompter/assets/129811700/8574960a-f0d6-471e-9044-6e939cc55421">

![screencapture-127-0-0-1-7860-2023-06-06-01_20_25](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/129811700/2d158ece-1071-4d71-9a42-98154024f42b)

https://anonfiles.com/PcRav0vbz1/err_txt
debug text file
use 2  lora and output it with the Hires.fix option, the KeyError message is output and it stops.

<img width="1920" alt="006" src="https://github.com/hako-mikan/sd-webui-regional-prompter/assets/129811700/05619456-5e82-4d3e-81b3-0be3ade9fd1e">

![screencapture-127-0-0-1-7860-2023-06-06-01_24_11](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/129811700/e3f4ad91-374e-45b9-8e65-e1f9c966a9de)

![00009-1523760198-2girls,city,night,_,     power ,sharp teeth,symbol-shaped pupils,cross-shaped pupils,blue jacket,collared shirt,black necktie,bl](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/129811700/9b0166bc-d164-40ed-8c60-ec274996b766)

https://anonfiles.com/G4T4v4v4z8/ok_txt
debug text file
If you don't use the Hires.fix option, the output will be normal

https://civitai.com/models/50520
https://civitai.com/models/72585
https://civitai.com/models/72595
This is the model and lora used for testing

**Comment by anae-git** [comment]
Yes, I am seeing the same. I described mine at https://github.com/hako-mikan/sd-webui-regional-prompter/issues/135#issuecomment-1576031752

**Comment by AndrewLieber** [comment]
Same issue here

**Comment by b14nmzq** [comment]
Try downgrading the webUI version to 1.2.1.
I also got a key error after updating to 1.3.2, but after downgrading I was able to generate it again.
I think it is not even possible to generate with 1 LoRA and this extension. Perhaps Regional Promter is not yet compatible with v1.3.2.

**Comment by freebag668** [comment]
I checked that it works in the latest version

**Comment by freebag668** [comment]
I checked that it works in the latest version

**Comment by freebag668** [comment]
I checked that it works in the latest version
